### PR TITLE
Fix grammar issue

### LINKED ISSUE #982: Grammar issues in `consumerParameters` section
https://github.com/oceanprotocol/docs/blame/main/content/concepts/did-ddo.md#L135
> An object the defines required consumer input...

"the" should be "that"

---

https://github.com/oceanprotocol/docs/blob/main/content/concepts/did-ddo.md#consumer-parameters
The Consumer Parameters section makes heavy use of the word "you" and "your" which results in ambiguous sentences. Is "you" the publisher? or the consumer? compute provider?

### PR DESCRIPTION
**Pull request by akshay-ap**
Fixes #982 

Changes proposed in this PR:

- [x] Fix grammar issue
- [x] Fix ambiguity in text

### CODE REVIEW

File: content/concepts/did-ddo.md [hunk 1]
```diff
@@ -444,11 +444,12 @@ Example:
...
+Sometimes, the asset may need some input before downloading a dataset or running an algorithm.
 Examples:
 
-- You want to know the desired sampling interval of data in your dataset, before the user is going to download it. Your dataset URL is `https://example.com/mydata`. So you will define a field called `sampling`, ask the user to enter a value and then this parameter is going to be added to the URL of your dataset as query parameters: `https://example.com/mydata?sampling=10`
-- Before running an algorithm, you need to know how many iterations should it perform. You define a field called `iterations`, ask the user to enter a value and this parameter is stored in a specific location in your Computer-to-Data pod for the algorithm to read and use that value.
+- The dataset publisher wants to know the desired sampling interval of data before the buyer downloads it. Suppose the dataset URL is `https://example.com/mydata`. So the publisher will define a field called `sampling`, ask the buyer to enter a value, and then this parameter is going to be added to the URL of the published dataset as query parameters: `https://example.com/mydata?sampling=10`.
```
Comment (AnaLoznianu) [inline content/concepts/did-ddo.md:450]: Suggestion: "Sometimes, the asset needs additional input data before downloading a dataset or running an algorithm.
The publisher needs to know the sampling interval before the buyer downloads it. Suppose the dataset URL is `https://example.com/mydata'. The publisher then defines a field called `sampling` and asks the buyer to enter a value. This parameter is added to the URL of the published dataset as query parameters: `https://example.com/mydata?sampling=10`."

File: content/concepts/did-ddo.md [hunk 2]
```diff
@@ -444,11 +444,12 @@ Example:
...
+
+- Before running an algorithm, the algorithm needs to know how many iterations should it perform. In this case, the algorithm publisher defines a field called `iterations`. The buyer enters the value for the `iterations` parameter, and later this value is stored in a specific location in the Computer-to-Data pod for the algorithm to read and use that value.
```
Comment (AnaLoznianu) [inline content/concepts/did-ddo.md:452]: Suggestion: "An algorithm that needs to know how many iterations should perform. In this case, the algorithm publisher defines a field called `iterations`. The buyer needs to enter a value for the `iterations` parameter. Later, this value is stored in a specific location in the Computer-to-Data pod for the algorithm to read and use it."

**Review by AnaLoznianu — CHANGES_REQUESTED** [review]

File: content/concepts/did-ddo.md [hunk 3]
```diff
@@ -444,11 +444,12 @@ Example:
...
 Examples:
 
-- You want to know the desired sampling interval of data in your dataset, before the user is going to download it. Your dataset URL is `https://example.com/mydata`. So you will define a field called `sampling`, ask the user to enter a value and then this parameter is going to be added to the URL of your dataset as query parameters: `https://example.com/mydata?sampling=10`
-- Before running an algorithm, you need to know how many iterations should it perform. You define a field called `iterations`, ask the user to enter a value and this parameter is stored in a specific location in your Computer-to-Data pod for the algorithm to read and use that value.
+- Sometimes, the asset needs additional input data before downloading a dataset or running an algorithm. The publisher needs to know the sampling interval before the buyer downloads it. Suppose the dataset URL is `https://example.com/mydata`. The publisher then defines a field called sampling and asks the buyer to enter a value. This parameter is added to the URL of the published dataset as query parameters: `https://example.com/mydata?sampling=10`.
```
Comment (AnaLoznianu) [inline content/concepts/did-ddo.md:450]: @akshay-ap We have the sometimes sentence twice now. 
Screenshot 2022-05-17 at 10 56 50

Please adjust the first sentence with the suggestion I offered.
Also, please add the proper formatting for the 'sampling' parameter

**Review by AnaLoznianu — CHANGES_REQUESTED** [review]

**Comment by trentmc** [comment]
Note: I just fixed the spelling issue in "fix grammer issue". It kept nagging me 🤷‍♂️ :)

**Comment by akshay-ap** [comment]
> Note: I just fixed the spelling issue in "fix grammer issue". It kept nagging me man_shrugging :)

apologies for the inconvenience 😞

**Review by AnaLoznianu — APPROVED** [review]

**Pull request merged**